### PR TITLE
refactor: prettier format

### PR DESCRIPTION
**Pull request by sergiofv**

### CODE REVIEW

**Review by koepferd — APPROVED** [review]

File: src/components/call-to-action/call-to-action.scss [hunk 1]
```diff
@@ -132,18 +149,34 @@ img {
   font-family: inherit;
 }
 
-.btn2 {
+.btn.dark {
```
Comment (koepferd) [inline src/components/call-to-action/call-to-action.scss]: please also use bem -> .btn-dark

also for --mid & --light

File: src/components/accordion-tab/accordion-tab.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-  tag: 'accordion-component',
-  styleUrl: 'accordion-component.scss',
+  tag: 'accordion-tab',
+  styleUrl: 'accordion-tab.scss',
   shadow: true,
 })
-export class AccordionComponent {
-  @Prop() title: string;
+export class AccordionTab {
+  @Prop() tabTitle: string;
```
Comment (koepferd) [inline src/components/accordion-tab/accordion-tab.tsx]: I would not change this prop name.
In the scope the accordion tab it is still the title.

File: src/components/accordion-tab/accordion-tab.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+:host {
+  display: block;
+}
+
+details {
+  font-family: 'Open Sans', sans-serif;
+  width: 620px;
+}
+
+details > summary {
```
Comment (koepferd) [inline src/components/accordion-tab/accordion-tab.scss]: As discussed: wrong selector here. Please use the add the summary & p styling directly in the details.

File: src/components/call-to-action/call-to-action.tsx [hunk 4]
```diff
@@ -13,62 +27,76 @@ export class CallButton {
...
+  @Prop({ reflect: true, mutable: true }) footers: Footer[];
+
+  @State() isOpen: boolean | null;
+
+  @State() isIndex: number | 0;
+
+  testFooters = [
+    {
+      buttonLabel: 'Info',
+      buttonClassname: 'btn dark',
```
Comment (koepferd) [inline src/components/call-to-action/call-to-action.tsx]: you can go with buttonClass & class as per convention.

File: src/components/call-to-action/call-to-action.tsx [hunk 5]
```diff
@@ -80,28 +108,16 @@ export class CallButton {
...
             </div>
           </div>
           <div class='banner__half'>{<img src={this.imagePath} />}</div>
         </div>
-        <div>{this.footerOneToggled ? footerOne : null}</div>
-        <div>{this.footerTwoToggled ? footerTwo : null}</div>
+        <div>
+          {this.isOpen
+            ? renderFooter(this.footers[this.isIndex])
+            : renderFooter(null)}
```
Comment (koepferd) [inline src/components/call-to-action/call-to-action.tsx]: *we've talked about this already ;-)

**Review by koepferd — CHANGES_REQUESTED** [review]

**Comment by sergiofv** [comment]
New repo with all the changes specified and API fetch feature now available [here](https://github.com/sergiofv/homepage-call-to-action). Now as a stencil project (not as storybook story, as it was before).

**Pull request closed without merging**